packages/next-server/server/config.js

```diff
@@ -18,7 +18,9 @@ const defaultConfig = {
   onDemandEntries: {
     maxInactiveAge: 60 * 1000,
     pagesBufferLength: 2,
-    websocketPort: 0
+    websocketPort: 0,
+    websocketProxyPath: null,
```

Comment:
Should this instead just be taking `assetPrefix` into account instead?

Reply:
I don't think so - at least in my case what I'm trying to accomplish here is set a unique path which corresponds to a location block in my nginx config. This allows the `sw` request to successfully establish itself. 

Here's a snippet from my conf files so you can see what I am working with:

I am listening on `7001` and forwarding to `7002`, which I have set as my `websocketPort` locally.

```
location /hmr {
  proxy_http_version 1.1;
  proxy_set_header Upgrade $http_upgrade;
  proxy_set_header Connection "upgrade";
  proxy_pass http://espresso_hot_server;
}
...elsewhere
upstream espresso_hot_server {
  server 127.0.0.1:7002;
}
```